datadog/resource_datadog_metric_metadata.go

```diff
@@ -81,6 +81,14 @@ func resourceDatadogMetricMetadataCreate(ctx context.Context, d *schema.Resource
 	auth := providerConf.Auth
 
 	id, m := buildMetricMetadataStruct(d)
+
+	// Datadog API only has partial supports for the distribution type in metrics endpoints,
+	// and sending the distribution type field in the request payload will result in a 400 error.
+	// This is an interium workaround so that users can still initialize resources with distribution metric.
+	if m.GetType() == "distribution" {
+		m.SetType("")
```

Comment:
Will an empty string result in a null value in the request? This might get invalidated by the API

Reply:
@jmfurlott https://github.com/DataDog/datadog-api-client-go/blob/master/api/datadogV1/model_metric_metadata.go#L23-L24

the API client sets `omitempty` so it will be ignored